js/app.jsx

```diff
@@ -486,6 +486,26 @@ var Job = React.createClass({
       </ul>
     );
 
+    function slimTrace(str) {
+      var lines = [];
+      if (str) {
+        lines = str.split(/,|ÿ/)
+                   .map(val => { return val.trim(); })
+                   .map(val => {
+                    var matches = val.match(/\w*\.\w+\(\w+\.scala:\d+\)/i);
```

Comment:
I suspect the extra context is more noise than signal. The hard part is tracking down where a job comes from. Once you have a line number it's straightforward enough to understand (or guess) what causes the sink.